### PR TITLE
Update ucasthesis.cfg

### PR DESCRIPTION
**Pull request by GitatHub**
Though the official guide uses The University of CAS,
as in http://yjsgl.ucas.ac.cn/index.php/zh-cn/fenlei/bydb/76-2016-10-10-02-06-23,
the official website use University of CAS w/o "the"
and "In" should be "in" and "For" should be "for"

### CODE REVIEW

**Comment by mohuangrui** [comment]
Hello,

You have raised an important point! I think there are other 
mis-capitalized words too. The correct one should be:

A _d_issertation submitted to
the University of Chinese Academy of Sciences
in partial fulfillment of the requirements
for the degree of
(Master of Fluid Mechanics)

I totally agree with you that there are mistakes; however, shall we 
change this before the official guide makes any changes?
I'm not sure... Maybe you can send an email to the officials who are 
responsible for the standards. How do you think about that?

Thanks a lot,
Huangrui

**Comment by mohuangrui** [comment]
Hello again,

I just checked the templates of several US universities, and I found 
that the inconsistency of capitalization is not that "wrong", as shown 
in the following screen-shots:

Caltech has an inconsistency like that of CAS:


Stanford does the most "clever" trick...


I think UC, Berkeley is consistent but a little bit ugly:



Harvard may have the most consistent one:


Overall, if the name of a university starts with "University of ...", 
then the "the" is required, such as The University of UC, Berkley, The 
University of CAS. Otherwise, the "the" should be omitted, such as 
Harvard University, Stanford University...

So, I will not consider the changes pushed by you so far :D

Thanks and have a great day,
Huangrui

**Comment by GitatHub** [comment]
Hello, I agree with you in that the "the" should be kept in the English cover.
However, as to the capitalization, I saw some templates of other universities, and I am sure that some letters in the first word of the line should be not capitalized.
For example, 
Princeton, https://www.sharelatex.com/templates/52fb8c8f34a287a85245b494/v/1/pdf?inline=true&name=Princeton%20University%20Thesis
"of" and "in" are in the lower case
Harvard, https://www.sharelatex.com/templates/52fb8a0b33621a613683ec98/v/4/pdf?inline=true&name=Harvard%20PhD
"for" and "in" are in the lower case
UC, Berkeley, https://www.sharelatex.com/templates/52fb8c9833621a613683ecb2/v/1/pdf?inline=true&name=University%20of%20California%20Berkeley%20Thesis
"of" and "in" are in the lower case, and the "the" before the name of University is in a different line.
MIT, https://www.sharelatex.com/templates/52fb8ae634a287a85245b480/v/1/pdf?inline=true&name=MIT%20Thesis
"in" and "at" are in the lower case.

In summary, I think those officials in the UCAS must have made some mistakes when they were preparing the thesis guide. I guess the reason is that they were using the MS WORD and the capitalizing were made by the WORD autocorrect.

I will nevertheless try to change the styles in my branch,
and thanks for you excellent job in maintaining this template.
